README.md

```diff
@@ -11,6 +11,27 @@ For more details about the SDK APIs, please refer to the **Getting Started** gui
 ---
 ​
 # How to Run the Flutter Demo Application​
+
+## Prerequisites
```

Comment:
For all the software to be installed, do we reply on any specific version? Like does this work with Xcode 14, with latest gradle?

Reply:
For now, I don't see any issues with latest Xcode or Gradle, we are expecting it to work with the latest softwares.